server/services/auditService.ts

```diff
@@ -17,6 +17,7 @@ export enum Page {
   CaseNotes = 'CASE_NOTES',
   CsraHistory = 'CSRA_HISTORY',
   CsraReview = 'CSRA_REVIEW',
+  Goals = 'GOALS',
```

Comment:
One last thing (sorry!) 😬 
Based on the fact we've just changed the page names etc 👍 , can we change this please to:
```
VirtualCampusGoals = 'VC2_GOALS`,
```

Then I think we're golden 👍

Reply:
Updated - good spot

Reply:
Thanks